src/main/java/seedu/address/model/module/LectureDetails.java

```diff
@@ -34,7 +34,7 @@ public LectureDetails(String lectureDetails) {
      * Returns true if a given string is a valid lecture detail.
      */
     public static boolean areValidLectureDetails(String test) {
-        return test.matches(VALIDATION_REGEX);
+        return true;
```

Comment:
Is this a temporary fix?

Reply:
No it's permanent because there are no more restrictions I think

Reply:
I feel like this part we need to discuss more on telegram

Reply:
the initial code should remain, it detects whether the string starts with a whitespace or not